src/components/sbb-datepicker/sbb-datepicker.tsx

```diff
@@ -270,7 +254,22 @@ export class SbbDatepicker implements ComponentInterface {
     return this.format ? this.format(date) : this._dateAdapter.format(date);
   }
 
+  private _setAriaLiveMessage(date: Date): void {
+    const ariaLiveFormatter = new Intl.DateTimeFormat(`${this._currentLanguage}-CH`, {
+      weekday: 'long',
+      year: 'numeric',
+      month: 'numeric',
+      day: 'numeric',
+    });
+
+    this._statusContainer.innerHTML = date
+      ? `${i18nDateChangedTo[this._currentLanguage]} ${ariaLiveFormatter.format(date)}`
+      : i18nNoDate[this._currentLanguage];
```

Comment:
Not sure how it would interfere with invalid state when you have form errors. would it then read "no date selected" and after something like "invalid date value" (or whatever)? So potentially it could be better to read only valid values (did it like so in time input)